utils/device.py

```diff
@@ -4,17 +4,32 @@
 
 # System
 import os
+import logging
 
 # Externals
 import tensorflow as tf
 
+# Locals
+import utils.distributed as dist
+
 def configure_session(intra_threads=32, inter_threads=2,
-                      blocktime=1, affinity='granularity=fine,compact,1,0',
```

Comment:
Note this changes the defaults. You'll need to take care to set these now via command line or env variables.